server/channels/app/platform/config.go

```diff
@@ -173,6 +173,8 @@ func (ps *PlatformService) regenerateClientConfig() {
 	clientConfig := config.GenerateClientConfig(ps.Config(), ps.telemetryId, ps.License())
 	limitedClientConfig := config.GenerateLimitedClientConfig(ps.Config(), ps.telemetryId, ps.License())
 
+	clientConfig["CWSMock"] = model.MockCWS
```

Comment:
Nit: Looks like this might fit into the `GenerateClientConfig` function a bit better, I see some cases where client config attributes are set similar to this, but only when there are special conditions or processes that occurs.

Reply:
Yep, good callout! Will remove.